warehouse/models/intermediate/gtfs/int_gtfs_schedule__stop_times_grouped.sql

```diff
@@ -1,9 +1,16 @@
-{{ config(materialized='table') }}
+{{
+    config(
+        materialized='incremental',
+        unique_key = 'key',
```

Comment:
I am slightly curious as to why these use a key-based strategy as opposed to just date partitions (@atvaccaro?), but I see this is the same as the parent so I assume it's fine.

Reply:
Originally I wanted to eventually support late-arriving data, and I don't think we'd benefit from using `insert_overwrite` so might as well stick with key.